package com.github.kimcore.koreanbots.entities.internal.exceptions

class LoopAlreadyRunningException : Exception("이미 루프가 실행 중입니다.")